Index: src/main/java/pe/edu/upc/demosi/dtos/LoginResponseDTO.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package pe.edu.upc.demosi.dtos;\r\n\r\npublic class LoginResponseDTO {\r\n    private final String token;\r\n    private final String correo;\r\n\r\n\r\n    public LoginResponseDTO(String token, String correo) {\r\n        this.token = token;\r\n        this.correo = correo;\r\n    }\r\n\r\n    public String getToken() {\r\n        return token;\r\n    }\r\n\r\n    public String getCorreo() {\r\n        return correo;\r\n    }\r\n}\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/pe/edu/upc/demosi/dtos/LoginResponseDTO.java b/src/main/java/pe/edu/upc/demosi/dtos/LoginResponseDTO.java
--- a/src/main/java/pe/edu/upc/demosi/dtos/LoginResponseDTO.java	(revision 72b9b6217f0e5947ea9d5a1cfcf9057483ffa6b0)
+++ b/src/main/java/pe/edu/upc/demosi/dtos/LoginResponseDTO.java	(date 1790401474075)
@@ -1,8 +1,8 @@
 package pe.edu.upc.demosi.dtos;
 
 public class LoginResponseDTO {
-    private final String token;
-    private final String correo;
+    private String token;
+    private String correo;
 
 
     public LoginResponseDTO(String token, String correo) {
